src/hxmath/math/Matrix2x2.hx

```diff
@@ -150,8 +150,8 @@ abstract Matrix2x2(Matrix2x2Type) from Matrix2x2Type to Matrix2x2Type
     {
         return new Matrix2x2(
             m.a * n.a + m.c * n.b,  // p_00 = m_i0 * n_0i
-            m.a * n.c + m.c * n.d,  // p_10 = m_i0 * n_1i
-            m.b * n.a + m.d * n.b,  // p_01 = m_i1 * n_0i
+            m.b * n.a + m.d * n.b,  // p_10 = m_i1 * n_0i
```

Comment:
Yikes, I re-read this a few times and realized what happened. Nice find! I have a request: can you split the Matrix2x2 fix (and any other non-test fixes) out into another PR? I'd like to have a revision where the functionality is fixed and running against existing tests before changing the tests.

Reply:
Done. I didn't check those pull requests against existing tests, but if I recall correctly they should pass. The matrix test used an identity matrix to test (so it didn't matter if it was transposed), and the vector test rotated around the origin.